index.js

```diff
@@ -281,7 +281,17 @@ function redisStore(args) {
       if (err) {
         return cb && cb(err);
       }
-      conn.del(key, handleResponse(conn, cb));
+
+      if (Object.prototype.toString.call(key) === '[object Array]') {
```

Comment:
The `sol-redis-pool` module now promisifies the underlying redis connections using `bluebird`. So, every command has an Async alternative, e.g. `setAsync()`, `setexAsync()`, `delAsync()`, etc...  It should be relatively painless to refactor all the methods to return a promise or accept a callback -- especially with bluebird's `.asCallback` method.